sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskService.java

```diff
@@ -863,6 +864,17 @@ public List<Selection<?>> getJurisdictionSelections(TaskQueryContext qc) {
 					environmentService.inJurisdictionOrOwned(new EnvironmentQueryContext(cb, qc.getQuery(), joins.getEnvironmentJoins())))));
 	}
 
+	private Predicate getObjectSubquery(TaskQueryContext qc, CriteriaBuilder cb, Function<TaskJoins, Predicate> jurisdictionPredicate) {
```

Comment:
please delete this methods because it's not used

Reply:
removed